libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -238,6 +242,8 @@ struct Component {
     33: optional set<string> releaseIds,
 
     35: optional set<string> mainLicenseIds,        //Aggregate of release main licenses
+    36: optional set<string> mainLicenseNames,
```

Comment:
I thought I've successfully deleted this field in #135. Why are you adding it back?